src/test/java/org/junit/validator/AnnotationValidatorFactoryTest.java

```diff
@@ -20,7 +20,7 @@ public void createAnnotationValidator() {
     }
 
     @ValidateWith(value = Validator.class)
-    public static class SampleTestWithValidator {
+    public @interface SampleTestWithValidator {
```

Comment:
nit: maybe rename "SampleTestWithValidator" to "AnnotationWithValidator" or "SampleAnnotationWithValidator"?

(same thing below)